src/composables/useUrlParams.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * A composable function that provides utilities for managing URL parameters.
+ */
+export const useUrlParams = () => {
+  /**
+   * Retrieves the current URL parameters as an object.
+   * @returns An object containing the URL parameters as key-value pairs.
+   */
+  const getUrlParams = (): Record<string, string> => {
```

Comment:
Vue router doesn't have some get and set method by default?

Reply:
My mistake here.
Since I switched to the route-based navigation for the tabs, we no longer need the composable.
Removed.